### PR TITLE
Update request-new-package.yml - Add checkbox to confirm user will ma…

### PR DESCRIPTION
**Pull request by Justinzobel**
…intain it

### CODE REVIEW

**Comment by joebonrichie** [comment]
So this is optional, just for clarification?

If this is checked and has the approved label we may be able to do some github trickery to auto assign the user.

**Comment by Justinzobel** [comment]
Yes, optional tick the box. We're more likely to approve something if the user themselves is willing to maintain it, right?

**Comment by joebonrichie** [comment]
Makes sense to me

**Comment by HarveyDevel** [comment]
Minor nit, I would use softer language. Something along the lines of "Would you like to be the maintainer for this package?" but otherwise nothing to add.

File: .github/ISSUE_TEMPLATE/request-new-package.yml
Comment (TraceyC77) [inline .github/ISSUE_TEMPLATE/request-new-package.yml:14]: The indentation of your inserted lines do not match the others. As a result, the template is invalid. Take a look at https://github.com/getsolus/packages/blob/a782116016fe1e5c38fa60353e0c705fc0fbdc9b/.github/ISSUE_TEMPLATE/request-new-package.yml and you'll see an error.

Once the indentation is fixed, this is good to go.

**Review by TraceyC77 — CHANGES_REQUESTED** [review]
Indentation needs to be corrected.

File: .github/ISSUE_TEMPLATE/request-new-package.yml
Comment (Justinzobel) [reply .github/ISSUE_TEMPLATE/request-new-package.yml]: Should now be fixed.

**Review by TraceyC77 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
